frontend/amundsen_application/static/js/ducks/search/api/v0.ts

```diff
@@ -44,13 +50,18 @@ export function searchResource(
   if (
     (resource === ResourceType.dashboard && !indexDashboardsEnabled()) ||
     (resource === ResourceType.user &&
-      (!indexUsersEnabled() || term.length === 0))
+      (!indexUsersEnabled() || term.length === 0)) ||
+    (resource === ResourceType.feature && !indexFeaturesEnabled())
```

Comment:
I would probably create a temporal variable called isNotIndexedFeature or something like that.

In general, this branch condition is getting a bit complicated to read, anything we can do to simplify would go a long way.